pageserver/src/tui_event.rs

```diff
@@ -53,6 +53,8 @@ impl Events {
             let ignore_exit_key = ignore_exit_key.clone();
```

Comment:
I think `disable_exit_key` and `enable_exit_key` aren't used anywhere. Can we remove this variable?

Reply:
Let's push tui fixes to another PR.